mods/seasonal/random-teams.js

```diff
@@ -41,6 +41,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 			// Nature needs to be a valid nature with the first letter capitalized ex: Modest
 			*/
 			// Please keep sets organized alphabetically based on staff member name!
+			'5gen': {
```

Comment:
2 < 5 so it should go first imo

Reply:
>> ["2", "5", "5", "2"].sort()
<< ["2", "2", "5", "5"]
no

Reply:
wait

Reply:
okay true lmao fixing

Reply:
~~when you fail to read orderings properly~~